### ISSUE TITLE
Find session flow is really not accessible

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#309621]
Testing #309100

* All it says is "Type to search entity" - what am I searching?
* When you type, it doesn't say "X results" or anything like that
* inside of the list, ctrl+F doesn't work (is there another keybinding to search?) this is important for keyboard users

I'm putting this all in one issue because I think it probably needs a full "accessibility pass". 

<img width="316" height="338" alt="Image" src="https://github.com/user-attachments/assets/2bf41395-4fa2-465e-a0d5-60d637e68596"/>

<img width="308" height="179" alt="Image" src="https://github.com/user-attachments/assets/0a5ab680-1300-4d94-8530-978de98b5c44"/>
